webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -161,7 +161,7 @@ export const stringIsPositiveNumber = value => {
  * @param {String} version - foreman short version
  */
 export const getManualURL = version =>
-  `https://theforeman.org/manuals/${version}/index.html`;
+  `/links/manual`;
```

Comment:
There's a `foremanUrl` helper which actually gets the correct URLs to Foreman since they're not guaranteed to be at `/`.